server/services/doc.js

```diff
@@ -0,0 +1,177 @@
+//  Defines various services for the role object
+
+const Sequelize = require('sequelize');
+// Require sequelize from the connection settings
+const sequelize = require('../../settings/connect');
+const jwt = require('jsonwebtoken');
+
+const secret = process.env.SECRET || 'documentmanagement';
+
+// Call the doc model and specify the arguments.
+const Doc = require('../../app/models/doc')(sequelize, Sequelize);
+const User = require('../../app/models/user')(sequelize, Sequelize);
+
+sequelize.sync({});
+
+/**
+ * Create a new document
+ * @param {object} req
+ * @param {function} res // Callback
+ * @returns {boolean} true if created, false otherwise.
+ */
+module.exports.createDocument = (req, res) => {
```

Comment:
why not create an object first and then use module.exports on the object as a whole